pjsip-apps/src/pjsua/pjsua_app.c

```diff
@@ -696,11 +744,37 @@ static void on_buddy_evsub_state(pjsua_buddy_id buddy_id,
 
 }
 
+static void on_buddy_evsub_blf_state(pjsua_buddy_id buddy_id,
+                 pjsip_evsub *sub,
+                 pjsip_event *event)
+{
+    char event_info[80];
```

Comment:
Where does the magic number come from? Please use a constant.

Reply:
For sample apps (i.e. which is not part of the PJSIP core library code), we are okay with using hardcoded buffer, as long as:
- it is of sufficient length for most typical use cases
- it doesn't introduce risk of buffer overflow